executor/bitmap.h

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+#include <fcntl.h>
+#include <stdio.h>
+
+#if GOOS_linux || GOOS_freebsd || GOOS_netbsd || GOOS_openbsd || GOOS_akaros
+#include <sys/mman.h>
```

Comment:
I am a little confused here.
I have added sys/mman.h for mmap here, but building executor for netbsd failed.
Throwing out the error that PROT_READ and MAP_PRIVATE is not declared.
When I remove the #if line, it works.

Reply:
That's not the netbsd build is failing, that's fuchsia build (parallel make output is confusing and is barely useful).
Try:
```
$ syz-big-env make executor TARGETOS=netbsd
// passes
$ syz-big-env make executor TARGETOS=fuchsia
// fails
```

See this for how to install syz-env/syz-big-env:
https://github.com/google/syzkaller/blob/master/docs/contributing.md#using-syz-env

Reply:
It seems that fuchsia doesn't have "mmap" and doesn't use shared memory to write out coverage. So, the fuchsia build is failed. Should I add `#if SYZ_EXECUTOR_USES_SHMEM` for init_bitmap and coverage_filter?  And executor for fuchsia would not have coverage filter.

Reply:
Yes, it's reasonable to make this not work for fuchsia. If any fuchsia users will be interested, they may port it later.